core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -177,6 +181,95 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
     blockManager.stop()
   }
 
+  private def setupBlockManagerMasterWithBlocks(withLost: Boolean): Unit = {
```

Comment:
can we add comment to explain what `withLost` means?

Reply:
ok sure.